pokemongo_bot/cell_workers/pokemon_transfer_worker.py

```diff
@@ -44,6 +44,10 @@ def work(self):
                         logger.log("Keep {} best {}, based on {}".format(len(best_pokemons),
                                                                          pokemon_name,
                                                                          order_criteria), "green")
+                        for best_pokemon in best_pokemons:
+                            logger.log("{} [CP {}] [Potential {}]".format(pokemon_name,
+                                                                          best_pokemon['cp'],
+                                                                          best_pokemon['iv']), 'green')
```

Comment:
@douglascamata Yeah, it is here for testing. To be sure that we keep needed pokemons and release not needed. We can remove it after testing.